python/ray/services.py

```diff
@@ -28,6 +24,14 @@
 import ray.local_scheduler
 import ray.plasma
 
+from ray.tempfile_services import (
+    set_temp_root, get_temp_root, get_logs_dir_path, get_ipython_notebook_path,
+    get_raylet_socket_name, get_temp_redis_config_path, new_redis_log_file,
+    new_raylet_log_file, new_local_scheduler_log_file, new_webui_log_file,
+    new_worker_log_file, new_log_monitor_log_file,
+    new_global_scheduler_log_file, new_plasma_store_log_file,
+    new_plasma_manager_log_file, new_monitor_log_file)
```

Comment:
Nit, I prefer putting each of these items in one line, and sort them alphabetically. that would be more readable and maintainable.